client/components/edit/children/__snapshots__/ChildList.test.tsx.snap

```diff
@@ -152,12 +167,15 @@ exports[`ChildList renders using ajax-loaded root data 1`] = `
       pageSize={10}
       pid=""
     >
-      <ul>
+      <ul
+        className="childlist"
+      >
         <li
           key="_child_foo:124"
         >
           <Component
             initialTitle="hello"
+            parentPid=""
```

Comment:
Just looking at this line a dozen times in this test makes me wonder if this should be the default value if not included?

Reply:
The ChildList defaults pid to blank string... and then it passes down that value as parentPid to all of its children. We could default parentPid to blank string in the Child component, but we'd then have to write more complex logic to omit the attribute when building the ChildList, which I don't think has any real benefit apart from reducing the size of this test snapshot (and has the disadvantage of increasing code complexity).

In any case, I've gone ahead and added the missing whitespace to the CSS files.